test/core/utils/send-command-to-frame.js

```diff
@@ -81,7 +84,7 @@ describe('axe.utils.sendCommandToFrame', function () {
   });
 
   it('should respond once when no keepalive', function (done) {
-    var number = 2;
+    var number = 1;
```

Comment:
I don't think that code would ever have triggered because the timeout below it was set to a very low value. I could revert this and put it into a separate PR.